database/migrations/2024_01_03_145604_create_enrollments_table.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateEnrollmentsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('enrollments', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
```

Comment:
Instead of modifying the table on line 31, this line can just be `$table->foreignId('user_id')`. Laravel uses convention to know that it is referencing the user table and id field.